lingetic-spring-backend/Dockerfile

```
@@ -15,15 +15,15 @@ COPY src/main src/main
 
 RUN --mount=type=secret,id=SENTRY_AUTH_TOKEN \
     export SENTRY_AUTH_TOKEN=$(cat /run/secrets/SENTRY_AUTH_TOKEN) && \
-    ./gradlew build -x test --no-daemon
+    ./gradlew nativeCompile -x test --no-daemon
 
-FROM amazoncorretto:23-jdk AS runtime
+FROM ubuntu:latest AS runtime
```

Hadolint (2.12.0)

[warning] Using latest is prone to errors if the image will ever update. Pin the version explicitly to a release tag

(DL3007)